cob_moveit_bringup/bin/add_robot.sh

```diff
@@ -1,2 +1,25 @@
 #!/bin/bash
-rosrun cob_moveit_config add_robot.sh "$@"
+
```

Comment:
These fixes should go into the  `cob_moveit_config` script.

Reply:
As an alternative, the script in `cob_moveit_config` should get removed.

Reply:
I need this here, as I don't have the `cob_robots` repo around - which is the reason for the additional arguments...removing the script from `cob_moveit_config` is ok, though...I'll provide a PR

Reply:
PR is https://github.com/ipa320/cob_robots/pull/729